frontend/src/metabase/components/ItemsTable/BaseItemsTable.styled.tsx

```diff
@@ -2,21 +2,28 @@
 import { css } from "@emotion/react";
 // eslint-disable-next-line no-restricted-imports
 import styled from "@emotion/styled";
-import type { HTMLAttributes } from "react";
+import cx from "classnames";
+import type { HTMLAttributes, TableHTMLAttributes } from "react";
 
 import EntityItem from "metabase/components/EntityItem";
 import IconButtonWrapper from "metabase/components/IconButtonWrapper";
 import Link from "metabase/core/components/Link";
 import AdminS from "metabase/css/admin.module.css";
-import type { TextProps } from "metabase/ui";
+import type { IconProps, TextProps } from "metabase/ui";
 import { FixedSizeIcon, Text } from "metabase/ui";
 
 import { RawMaybeLink } from "../Badge/Badge.styled";
 
 import type { ResponsiveProps } from "./utils";
 import { getContainerQuery } from "./utils";
 
-export const Table = styled.table<{ isInDragLayer?: boolean }>`
+type TableProps = TableHTMLAttributes<HTMLTableElement> & {
+  isInDragLayer?: boolean;
+};
+
+export const Table = styled((props: TableProps) => (
+  <table {...props} className={cx(AdminS.ContentTable, props.className)} />
```

Comment:
isn't this a different behaviour? this uses both ContentTable and the className passed, the old logic was defaulting to ContentTable

Reply:
@npretto good point, let me tweak this to `className={props.className ?? AdminS.ContentTable}` instead